index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Roboto' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="logo">My Logo</div>
+        <div id="menu-btn">
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+          <div class="menu-bar"></div>
+        </div>
+      </nav>
+    </header>
+    <div id="headline-section">
+      <h1>Hey There. I'm Charles.</h1>
+      <div id="introduction-text">I'm a Software Developer</div>
```

Comment:
- [ ] This is the second heading on the webpage, and it is important to use the proper tag for it which I believe is an `<h2>` tag. Using the `<h2>` tag will make the webpage easy for web crawlers to access and then rank the page properly.